Source/WebCore/page/DiagnosticLoggingKeys.h

```diff
@@ -120,7 +120,6 @@ class DiagnosticLoggingKeys {
     static String visuallyEmptyKey();
     static String pageContainsAtLeastOneMediaEngineKey();
     static String pageContainsMediaEngineKey();
-    static String pageHandlesWebGLContextLossKey();
```

Comment:
You can also remove:

- activeInForegroundTabKey
- activeInBackgroundTabOnlyKey
- inactiveKey